### PR TITLE
Django phonenumber field pinning

### LINKED ISSUE #338: django-phonenumber-field requirement too strict?
Hi,

I'm trying to implement a better payment solution in an existing setup. The customer is using [Oscar ecommerce](https://django-oscar.readthedocs.io/), the latest release has the following pin: `django-phonenumber-field<6.0.0,>=4.0.0`
This conflicts with the pinning of `django-phonenumber-field>=6.0.0` that django-payments uses.

Looking at the code in django-payments, I don't see anything that requires this minimal version: the model field is used and that's about it. The 'support for python3.10' that was added in django-phonenumber-field==6.0.0 are only changes to the test matrix and docs, no actual codechanges have been done. 

Would you consider a more lenient pinning, to f.i. 5.0.0? Happy to submit a PR, of course

FYI I'm trying to implement PSP Mollie using django-payments and Oscar. As being the maintainer of the [Mollie Python SDK](https://github.com/mollie/mollie-api-python/), also interested in releasing it as a general plugin to django-payments, and also supporting multiple payments methods through Mollie in the end (not only iDeal as suggested/requested in some places).

### PR DESCRIPTION
**Pull request by whyscream**
I tested `django-phonenumber-field[phonenumberslite]==5.0.0` with all configurations in tox.ini and it works with  django 3.2 and below. Version `5.1.0` works with all python/django combinations in tox.

Resolves #338

### CODE REVIEW

**Comment by WhyNotHugo** [comment]
5.1.0 adds support for Python 3.9 and Django 3.2, but it seems that no actual changes were necessary:

https://github.com/stefanfoulis/django-phonenumber-field/commit/4816c5000bc84f2c37698f125e26e85bb9dbe221

https://github.com/stefanfoulis/django-phonenumber-field/commit/07e3e0e930796372e7026e011dac787f579c1fe7

I'll let tests run here just in case, but will merge if everything is green.

**Review by WhyNotHugo — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -14,3 +14,4 @@ addopts =
   --color=yes
 testpaths = payments
 DJANGO_SETTINGS_MODULE = test_settings
+pythonpath = .
```
Comment (WhyNotHugo) [inline setup.cfg:17]: Why is this here?

Comment (whyscream) [reply setup.cfg:17]: This was need to resolve the problem in https://github.com/jazzband/django-payments/actions/runs/3783046529, where all test suites fail because django cannot find the settings file. I don't know it started, but this solves the problem in a clean way. If you have a better way to resolve it, please do.

Comment (whyscream) [reply setup.cfg:17]: @WhyNotHugo is this okay?

Comment (WhyNotHugo) [reply setup.cfg:17]: Yeah, it was previously not required and I can't figure out why now it is. I don't see anything in tox's changelog that would result in this being needed now.

**Comment by WhyNotHugo** [comment]
Thanks.
